core/src/main/java/com/oath/oak/InternalOakMap.java

```diff
@@ -1415,50 +1415,50 @@ public T next() {
             this.internalOakMap = internalOakMap;
         }
 
-        public Map.Entry<OakDetachedBuffer, OakDetachedBuffer> next() {
+        public Map.Entry<OakUnscopedBuffer, OakUnscopedBuffer> next() {
             advance(true);
-            return new AbstractMap.SimpleImmutableEntry<>(getKeyDetachedBuffer(ctx), getValueDetachedBuffer(ctx));
+            return new AbstractMap.SimpleImmutableEntry<>(getKeyUnscopedBuffer(ctx), getValueUnscopedBuffer(ctx));
         }
     }
 
-    class EntryStreamIterator extends Iter<Map.Entry<OakDetachedBuffer, OakDetachedBuffer>>
-            implements Map.Entry<OakDetachedBuffer, OakDetachedBuffer> {
+    class EntryStreamIterator extends Iter<Map.Entry<OakUnscopedBuffer, OakUnscopedBuffer>>
+            implements Map.Entry<OakUnscopedBuffer, OakUnscopedBuffer> {
 
-        private final OakDetachedReadBuffer<KeyBuffer> key = new OakDetachedReadBuffer<>(new KeyBuffer());
-        private final OakDetachedReadBuffer<ValueBuffer> value =
-                new OakDetachedReadBuffer<>(new ValueBuffer(valueOperator));
+        private final UnscopedBuffer<KeyBuffer> key = new UnscopedBuffer<>(new KeyBuffer());
```

Comment:
So there is no unscoped-read/write- buffer anymore?

Reply:
Unscoped write buffer never existed before.
Before, I added "read" to the unscoped buffer name. But I think it is redundant.
Don't you think?

Reply:
Unscoped buffer indeed exists for read only. As it is internally used name I am OK with leaving it like this. Maximum we will change it later.